config/nrfconnect/chip-module/Kconfig

```
@@ -88,6 +88,9 @@ config CHIP_DEBUG_SYMBOLS
 	help
 	  Enables building the application with debug symbols.
 
+config CHIP_MALLOC_SYS_HEAP
+	default y
```

```suggestion
	default y if !ARCH_POSIX
```
I think this is needed to fix native_posix tests.